### DISCUSSION TITLE
Standard Gross Charge for Drugs using pricing algorithm

### DISCUSSION THREAD

**Discussion by asvbarnes** [CMSgov/hospital-price-transparency#82] [Q&A] [answered]
For Drug formularies that contain Drugs (HCPCS, NDC, Unit of Measure, etc.) the hospital does not have a standard gross charge but rather a pricing algorithm based on actual dosage given (e.g. liquid or injectable). How would CMS except hospital to present standard gross charge as this is a dollar encoded column?

**Comment by RidenourK** [comment, marked as answer]
@asvbarnes. Thank you for your question. 

The gross charge is defined as the charge for an individual item or service that is reflected on a hospital’s chargemaster, absent any discounts.

**Comment by asvbarnes** [comment]
The charge appears as 0 (zero) in chargemaster for drugs as they are priced based on dosage given at the time of billing using a pricing formula in the PAS. However, CMS will not allow 0 (zero) in standard gross charge column. How would CMS recommend handling? Calculating lowest dosage price or leaving standard gross charge blank?

**Reply by RidenourK** [reply]
@asvbarnes. Thank you for your question. 

Standard charge means the regular rate established by the hospital for an item or service provided to a specific group of paying patients (45 CFR §180.20). Also at 45 CFR 180.20, we finalized the definition of gross charge, as a type of standard charge, to mean the charge for an individual item or service that is reflected on a hospital’s chargemaster, absent any discounts.  We further indicated that rule definition aligns with the Provider Reimbursement Manual Part 1 (PRM1)  “Charges refer to the regular rates established by the provider for services rendered to both beneficiaries and to other paying patients. Charges should be related consistently to the cost of the services and uniformly applied to all patients whether inpatient or outpatient. All patients’ charges used in the development of apportionment ratios should be recorded at the gross value; i.e., charges before the application of allowances and discounts deductions.” See further discussion at 84 FR 65540.

Hospitals are required to make a good faith effort to ensure that the standard charge information encoded in the machine-readable file is true, accurate, and complete as of the date indicated in the machine-readable file, which is required to be updated annually (once in a 12-month period). In the final rule we indicated we recognize that such data may, for various reasons, become outdated over the course of a 12-month period, but we also recognize that it may be burdensome for a hospital to continually update its standard charge information.

**Comment by AH-GJ** [comment]
In my opinion - you can take one of two approaches to satisfy the regulations repeated above. Either report the price of 1 unit for each drug assuming that is "the" dose; or estimate the average dose and price of each drug using dispense data collected over some time period. Having to report a single price where a regressive markup method is applied to determine a price does not seem to be contemplated in the regulations or response above. (for example: cost of 1 unit is $1.00 and markup is 1.5 plus overhead fee of $2.00 equals price of $3.50 and 2 units equals a price of $5.00).
